package.json

```diff
@@ -7,7 +7,7 @@
     "index.js"
   ],
   "scripts": {
-    "test": "grunt"
+    "jp-test": "mocha --compilers coffee:coffee-script"
```

Comment:
Good you noticed, this was a leftover from some development stage. No need to change test command, should be just as it was.